FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.h

```diff
@@ -40,7 +40,8 @@ should be defined by declaring a const structure of this type. */
 typedef struct xCOMMAND_LINE_INPUT
 {
 	const char * const pcCommand;				/* The command that causes pxCommandInterpreter to be executed.  For example "help".  Must be all lower case. */
-	const char * const pcHelpString;			/* String that describes how to use the command.  Should start with the command itself, and end with "\r\n".  For example "help: Returns a list of all the commands\r\n". */
+	const char * const pcDescriptionString;			/* String that describes how to use the command.  Should start with the command itself, and end with "\r\n".  For example "help: Returns a list of all the commands\r\n". */

```

Comment:
What is the difference between the new pcDescriptionString and the pre-existing pcHelpString?

Reply:
pcHelpString - a short description of the command when displaying all commands
pcDescriptionString - full description of the command when requesting "help command"

Reply:
This is a nice improvement to me.

Reply:
This "full description" feature should be optional can leave developer to configure.